docs/reference/sql_adapter.rst

```diff
@@ -300,17 +300,20 @@ construct is mapped to PostgreSQL schema:
 
 - Aliases are not mapped to PostgreSQL schema.
 
-- Globals are mapped to connection settings, prefixed with ``global``.
-  For example, a ``global default::username: str`` can be set using:
-  
-  .. code-block:: sql
+.. versionchanged:: 6.0
 
-      SET "global default::username" TO 'Tom'``.
+    - Globals are mapped to connection settings, prefixed with ``global``.
+      For example, a ``global default::username: str`` can be accessed using:
 
-- Access policies are applied to object type tables when setting
-  ``apply_access_policies_pg`` is set to ``true``.
+      .. code-block:: sql
 
-- Mutation rewrites and triggers are applied to all DML commands.
+          SET "global default::username" TO 'Tom'``;
+          SHOW "global default::username";
+
+    - Access policies are applied to object type tables when setting
+      ``apply_access_policies_pg`` is set to ``true``.
+
+    - Mutation rewrites and triggers are applied to all DML commands.
 
 
 DML commands
```

Comment:
@scotttrinh How can I mark these three whole sections as "added in EdgeDB 6.0"?

Reply:
Like [this](https://github.com/edgedb/edgedb/blob/0e8d80100e3d7c104747f336a8b19991d6a607c3/docs/reference/edgeql/casts.rst?plain=1#L37)

```rst
.. versionadded:: 6.0
```

Reply:
Oh, you mean the whole sections, 🤔

Reply:
I think you'll just need to mark each section separately? @jaclarke is there any better way to do this?

Reply:
But wouldn't that leave the section header in the old versions?

Reply:
I guess I could move the sections into separate pages.

Reply:
> But wouldn't that leave the section header in the old versions?

I don't think so? I think it shows as a half-hidden field that you can expand so you can see the new feature even from older versions which might make you want to upgrade. Worth experimenting and seeing how it renders.

Reply:
How do I test that? I see that I can `make docs` to get the output HTML in `build/html`. But how to I set the `versionadded` variable?

Reply:
Ahh, you'll need to get the docs infra setup locally via the https://github.com/edgedb/edgedb.com repo, which I think is missing the setup instructions about linking to your local repos 🤔